client/components/Cart.js

```diff
@@ -36,16 +36,19 @@ export class Cart extends Component {
       },
     };
     await this.props.updateCart(this.props.userId, itemUpdatedInCart);
+    // JOE_CR: Why refetch here?
     await this.props.fetchCartProducts(this.props.userId);
   }
 
   async removeItem(userId, productId) {
     await this.props.removeProductFromCart(userId, productId);
+    // JOE_CR: Why refetch here?
```

Comment:
@00-joe-js Refetching bc this reloads the products in the user's cart once a change is made (in these cases, update/delete). Tried playing around with componentDidUpdate but couldnt get it to work correctly